dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -1088,13 +1088,18 @@ public ArrayCreation(TypeMirror type, List<Node> dimensions, List<Node> initiali
             return initializers;
         }
 
+        @SuppressWarnings("EqualsIncompatibleType")
```

Comment:
@smillst This seems like a real problem: classes `Node` and `Receiver` are in no relationship, so the two loops in the body will never return true.
Do you know what the intended behavior here is?

Reply:
dimensions and initializers should be FlowExpressions.Receivers.  I'll fix it in a separate pull request.

Reply:
See PR #1758